README.md

```diff
@@ -25,11 +47,10 @@ Doing it this way accomplishes two things:
 3. Allows the possibility of merging back to original, or merging from original if or when the need arises
 4. Prevents lock-out by giving administrative access to more than one person, increases transferability
 
-## Misc
+### Misc
```

Comment:
Should probably rename this to `Contact us`. (And I still have to learn how to securely sign into freenode.)

Reply:
Renamed it. To me, it would be better if this section was merged with the `Support` section at some point.